packages/react-context-selector/src/useContextSelector.ts

```diff
@@ -78,3 +89,18 @@ export const useContextSelector = <Value, SelectedValue>(
 
   return state[1] as SelectedValue;
 };
+
+// Object.is polyfill: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is#polyfill
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const objectIs = (x: any, y: any) => {
```

Comment:
Can we reuse there a React implementation? It will fallback to `Object.is` if it's present:
https://github.com/facebook/react/blob/9198a5cec0936a21a5ba194a22fcbac03eba5d1d/packages/shared/objectIs.js

Reply:
copy+paste